app/views/admin/features_enabled.html.erb

```diff
@@ -28,6 +28,17 @@
                            'Help URL', "This is the URL for the external help e.g #{Seek::Config.default_external_help_url}.") %>
   </div>
 
+  <%= admin_checkbox_setting(:project_single_page_enabled, 1, Seek::Config.project_single_page_enabled,
+                             "Single page enabled", "The single page UI for #{t(:project).pluralize}", :onchange=>toggle_appear_javascript('single_page_folders')) %>
+  <div id="single_page_folders" class="additional_settings" style="<%= show_or_hide_block Seek::Config.project_single_page_enabled -%>">
+    <%= admin_checkbox_setting(:project_single_page_folders_enabled, 1, Seek::Config.project_single_page_folders_enabled,
+                               "Project folders enabled","If set, the folders of the projects will be shown in the tree view.") %>
+  </div>
+  <%= admin_checkbox_setting(:isa_json_compliance_enabled, 1, Seek::Config.isa_json_compliance_enabled,
```

Comment:
Something to maybe think about:
Would be nice if the user is only able to select this option if both 'Single Page enabled' and 'ISA enabled' are checked.

Reply:
Good suggestion, we will look into some javascript magic to do this